src/utils/render-outro-message.ts

```diff
@@ -12,23 +12,15 @@ export async function renderOutroMessage(options: Options) {
   ${chalk.dim("cd")} ${options.project}
   `;
 
-  if (options.extensions.includes("hardhat") || options.extensions.includes("foundry")) {
+  if (
+    options.extensions.includes(SOLIDITY_FRAMEWORKS.HARDHAT) ||
```

Comment:
maybe `===` again?

Reply:
uh, no, sorry

Reply:
I was thinking this for while, ideally I think we should rename `extensions`in `BaseOptions` to `solidityFramework`and isntead of it being an array it should be just a single value. 

I agree with this [line from doc](https://arc.net/l/quote/fdlhjdpl) but I think we are planning to / have bit diverged form it , 

Currently we have 3 different names for same thing SolidityFrameWorks(at type level and flag) === core extensions (in theory and while speaking) === extensions (the way the code and folder structure is structured)  

I think we are playing to no more add any core extensions(extensions that live in this `create-eth` repo)? So maybe in future rename `extensions` => `solidityFramework` ? 

```diff
// types.ts line 16: 
type FinalMergedOptions = {
  project: string | null;
  install: boolean | null;
  dev: boolean;
  externalExtension: ExternalExtension | null;
- extensions: Extension[];
+ solidityFramework: SolidityFramework
};
```

Reply:
> So maybe in future rename extensions => solidityFramework?

I agree, it will be much easier to understand and write code

Reply:
I'm 100% in with this change!

I think we originally planned for a more complex system. IMO we should keep simplifying / cleaning stuff as in #45.